backend/docker-compose.yml

```
@@ -0,0 +1,26 @@
+version: '3.8'
+services:
+  db:
+    container_name: postgresdb
+    image: postgres
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_DB: potter_db_api_development
+    ports:
+      - "5432:5432"
+    restart: always
+
+  web:
+    build: .
+    container_name: railsapp
+    ports:
+      - "3000:3000"
+    depends_on:
+      - db
+    env_file:
+      - .env  # Specify the .env file to load environment variables
```

```suggestion
    environment:
      DB_HOST: db
      POSTGRES_USER: postgres
      POSTGRES_PASSWORD: postgres
```
...and setting the environment variables here. Then we don't need any `.env` files.

---

Ok